### ISSUE TITLE
Forget policy is ignored

### ISSUE THREAD

**Issue by yurkoy** [creativeprojects/resticprofile#9]
resticprofile does not seem to pass retention policy to restic when executing `forget` command:

```txt
PS C:\Users\yuriy> resticprofile profiles
2020/07/13 11:34:12 resticprofile 0.8.2 compiled with go1.14.4
2020/07/13 11:34:12 using configuration file: c:\restic\profiles.yaml
2020/07/13 11:34:12 memory available: 38095MB

Profiles available:
  default:    (forget, retention)
  lightroom:  (backup, snapshots)
  photo:      (backup, snapshots)
  stdin:      (backup, snapshots)

Groups available:
  photo-backup:  photo, lightroom

PS C:\Users\yuriy> resticprofile -n default -v forget
2020/07/13 11:34:16 resticprofile 0.8.2 compiled with go1.14.4
2020/07/13 11:34:16 using configuration file: c:\restic\profiles.yaml
2020/07/13 11:34:16 memory available: 38092MB
2020/07/13 11:34:16 Setting priority class BELOW_NORMAL
2020/07/13 11:34:16 files in configuration are relative to 'c:\restic'
2020/07/13 11:34:16 Profile 'default': Starting 'forget'
2020/07/13 11:34:16 Starting command: C:\restic\bin\restic.exe forget --verbose --repo "c:\mnt\WD My Passport Slim 2TB\backup\restic-repo" --password-file c:\restic\restic-password.txt
repository fd96a456 opened successfully, password is correct
no policy was specified, no snapshots will be removed
2020/07/13 11:34:17 Profile 'default': Finished 'forget'
PS C:\Users\yuriy>
PS C:\Users\yuriy> cat C:\restic\profiles.yaml
global:
    default-command: version
    initialize: false
    priority: low

groups:
    photo-backup:
    - photo
    - lightroom

default:
    initialize: false
    password-file: restic-password.txt
    repository: "c:\\mnt\\WD My Passport Slim 2TB\\backup\\restic-repo"
    verbose: true
    retention:
        after-backup: true
        keep-hourly: 24
        keep-daily: 180
        keep-monthly: 12
        keep-yearly: 3
        keep-tag: forever
        prune: false
    forget:
        keep-hourly: 24
        keep-daily: 180
        keep-monthly: 12
        keep-yearly: 3
        keep-tag: forever

photo:
    inherit: default
    backup:
        source: "d:\\photo"
        tag: photo
        use-windows-vss: true
    snapshots:
        tag: photo

lightroom:
    inherit: default
    backup:
        source: "c:\\users\\yuriy\\lightroom"
        iexclude: "c:\\users\\yuriy\\lightroom\\*.lrdata"
        tag: lightroom
        use-windows-vss: true
    snapshots:
        tag: lightroom

stdin:
    backup:
        stdin: true
        stdin-filename: stdin-test
        tag:
        - stdin
    inherit: default
    snapshots:
        tag:
        - stdin
PS C:\Users\yuriy>
```

**Comment by creativeprojects** [comment]
Hmmm I should look into that today, thanks for reporting it

**Comment by creativeprojects** [comment]
Fixed in release 0.8.3

**Comment by yurkoy** [comment]
Thank you, it works for me with 0.8.3!
